scripts/patch_hosts.py

```diff
@@ -49,7 +49,7 @@ def path(self) -> Path:
             old_hosts = "/private/etc/hosts"
             p = new_hosts if os.path.exists(new_hosts) else old_hosts
         elif Platform.windows():
-            p = "C:\Windows\System32\drivers\etc\hosts"
+            p = r"C:\Windows\System32\drivers\etc\hosts"
```

Comment:
I don't think using `r""` here is correct.

Reply:
reverted

Reply:
reverted